lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -1032,3 +1033,98 @@ class TurkishAssetPickerTextDelegate extends AssetPickerTextDelegate {
   @override
   String get sUnitAssetCountLabel => 'Sayı';
 }
+
+/// [AssetPickerTextDelegate] implements with Korean.
+/// 한국어 현지화
+class KoreanAssetPickerTextDelegate extends AssetPickerTextDelegate {
+  const KoreanAssetPickerTextDelegate();
+
+  @override
+  String get languageCode => 'kr';
```

Comment:
`kr` stands for *Kanuri*? This should be `ko`?

Reply:
In Korea, I used 'kr' more often and used the code, but I changed it to 'ko' following the review!

Reply:
I'm not sure you'll get the Korean localization if you put `Locale('kr')` in Flutter. Can you identify that?

Reply:
Yes, but based on the comment LeGoffMael wrote, it looks better to apply it as 'ko'